docs/core/deploying/index.md

```diff
@@ -60,6 +64,28 @@ It also has a number of disadvantages:
 
 - Deploying numerous self-contained .NET Core apps to a system can consume significant amounts of disk space, since each app duplicates .NET Core files.
 
+## Framework-dependent executables (FDE)
+
+Starting with .NET Core 2.2, you deploy your app as an FDE, and any required thrid-party dependencies. You don't have to deploy .NET Core, since your app will use the version of .NET Core that's installed on the target system.
```

Comment:
you deploy -> you can deploy?

You don't have to deploy .NET Core -> You don't have to deploy .NET Core with your application? This sentence is slightly confusing since you do need to deploy the .NET Core runtime on the target system in order for this to work no?

Reply:
I used the wording from the other sections. I'll adjust both.